.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contributing
+This page contains the contributing guidelines.
+
+## Setup
+### prerequisite
+- Node.js installed. 
+ _Note: Node 6 or greater would be better for "best results"._
+- Package manager - [yarn](https://yarnpkg.com/en/docs/getting-started) or [npm](https://docs.npmjs.com/)
+
+
+
+### Project Setup in your local machine
+-   Fork the **just** repo at [https://github.com/microsoft/just](https://github.com/microsoft/just).
+-   `git clone <your-clone-url> && cd just`
+
+### Dependency Installation
+
+- Using npm
```

Comment:
This section about npm should probably not be there

Reply:
done